src/test/groovy/org/prebid/server/functional/util/AllureReporter.groovy

```diff
@@ -253,4 +272,46 @@ class AllureReporter extends AbstractRunListener implements IGlobalExtension {
         lifecycle.stopTestCase(uuid)
         lifecycle.writeTestCase(uuid)
     }
+
+
+    private void writeBlocks(FeatureInfo feature, IterationInfo iteration) {
+        def blockText = StringUtils.EMPTY
+        for (BlockInfo block : feature.blocks) {
+            if (!isEmptyOrContainsOnlyEmptyStrings(block.texts)) {
+                def size = block.texts.size()
+                for (int i = 0; i < size; i++) {
+                    if (iteration != null) {
+                        blockText = generationParams(block.texts[i], feature.dataVariables, iteration)
+                    }
+                    def kind = i == 0 ? block.kind.name() : "and"
+                    writeStep(kind, blockText)
+                }
+            } else {
+                writeStep(block.kind.name(), "-----")
+            }
+        }
+    }
+
+    private String generationParams(String input,
+                                    final List<String> dataVariables,
+                                    final IterationInfo iteration) {
+        def tempFeature = new FeatureInfo()
```

Comment:
I guess you can make it without declaring and initializing tempFeature variable here using new FeatureInfo().tap { }